[gas] add script to generate gas schedule update proposal

vgao1996 opened the pull request:
This adds a script to generate the gas schedule update proposal.

To run the script:
```
cargo run -p aptos-gas -o <output path>
```

<!-- Reviewable:start -->
- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/aptos-labs/aptos-core/4471)
<!-- Reviewable:end -->

vgao1996 commented:
Hmm... I probably messed up the commit history... Let me fix it

davidiw commented on aptos-move/aptos-gas/src/gen.rs:
```diff
@@ -0,0 +1,125 @@
+    // TODO: use relative path here
+    pack.add_local_dep("AptosFramework", &aptos_framework_path().to_string_lossy());
+
+    pack.write_to_disk(args.output.as_deref().unwrap_or("./proposal"))?;
```
probably `proposal.move`?

vgao1996 replied:
It's a move package.

davidiw reviewed (approved):
might recommend just shoving this into aptos-gas rather than making another crate.

vgao1996 commented:
@davidiw it's already in the captos-gas` crate. Notice that the new crate I added is a helper to generate Move packages, which I moved out of `e2e-move-tests` due to a cyclic dependency issue.

davidiw commented:
I ... am confused :) let's get this in!
